### ISSUE TITLE
moose-tools cannot be unloaded but provides a crappy gnuplot

### ISSUE THREAD

**Issue by dschwen** [idaholab/package_builder#215]
see title. I'd rather use the system gnuplot (which is 5.x) than the provided 4.6 gnuplot. Why are we shipping this anyways? I previously just unloaded the moose-tools module as it provided no value to me (beyond what I could install using the package manager). This is not possible anymore:

```
Unloading moose-tools
  ERROR: moose-tools cannot be unloaded due to a prereq.
    HINT: Might try "module unload moose-dev-clang" first.
```

**Comment by milljm** [comment]
Agreed. I'll add gnuplot to conda, instead of building it ourselves (and thus letting it get outdated)

**Comment by dschwen** [comment]
I see. My suggestion would be to more heavily rely on Linux package managers to provide those dependencies.

**Comment by dschwen** [comment]
Regarding using Conda for environment setup:

![image](https://user-images.githubusercontent.com/202302/68780194-268a2180-05f3-11ea-98ad-2c61f5d68198.png)

(yeah, I know, it is probably easier for you guys...)

**Comment by milljm** [comment]
Yep. Way easier said than done.
